resources/.eslintrc-es6.js

```diff
@@ -12,255 +12,21 @@ module.exports = {
-        "padded-blocks": ["error", "never"],                                                      //  require or disallow padding within blocks
-        "quote-props": ["error", "consistent-as-needed"],                                         //  require quotes around object literal property names
-        "quotes": ["error", "single"],                                                            //  enforce the consistent use of either backticks, double, or single quotes
-        "require-jsdoc": "off",                                                                   //  require JSDoc comments
-        "semi-spacing": "error",                                                                  //  enforce consistent spacing before and after semicolons
-        "semi": "error",                                                                          //  require or disallow semicolons instead of ASI
-        "sort-keys": "off",                                                                       //  require object keys to be sorted
-        "sort-vars": "off",                                                                       //  require variables within the same declaration block to be sorted
-        "space-before-blocks": "error",                                                           //  enforce consistent spacing before blocks
-        "space-before-function-paren": ["error", {                                                //  enforce consistent spacing before function definition opening parenthesis
-            "anonymous": "always",
-            "named": "never",
-            "asyncArrow": "ignore"
-        }],
-        "space-in-parens": "error",                                                               //  enforce consistent spacing inside parentheses
-        "space-infix-ops": "error",                                                               //  require spacing around infix operators
-        "space-unary-ops": "error",                                                               //  enforce consistent spacing before or after unary operators
-        "spaced-comment": "error",                                                                //  enforce consistent spacing after the SLASHSLASH or /* in a comment
-        "unicode-bom": "error",                                                                   //  require or disallow Unicode byte order mark (BOM)
-        "wrap-regex": "off",                                                                      //  require parenthesis around regex literals
-
         /**
-         * ECMASCRIPT 6
+         * ECMASCRIPT 2015
          *
-         * These rules relate to ES6, also known as ES2015
+         * These rules relate to ES2015, also known as ES6
          */
-        "arrow-body-style": "off",                    //  require braces around arrow function bodies
```

Comment:
Ah I see. I feel like it adds unnecessary syntax, especially when chaining a few things together. E.g this will error:
```
storyResponse
    .filter((story) => story.contentType === 'VIDEO')
    .map(someFunctionHere)
    .reduce((acc, current) => acc + current.someProperty, 0); 
```